Libraries/LibELF/DynamicLoader.cpp

```diff
@@ -273,6 +290,16 @@ void DynamicLoader::do_relocations()
             *patch_ptr = relocation.offset() - (u32)m_tls_segment_address.as_ptr() - *patch_ptr;
             break;
         }
+        case R_386_TLS_DTPMOD3: {
```

Comment:
If we do
```
set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```
and
```
-ftls-model=initial-exec
```
in the top level CMakeLists.txt, then  we get 1) PIC code for everyone (still works with statically linked :) and 2) Forced Initial Executable TLS model (which generates the R_386_TLS_TPOFF we were handling before with the non-PIE executable
https://docs.oracle.com/cd/E19120-01/open.solaris/819-0690/chapter8-20/index.html
https://docs.oracle.com/cd/E19120-01/open.solaris/819-0690/chapter8-1/index.html